CollAction/ValidationAttributes/RichTextRequiredAttribute.cs

```diff
@@ -0,0 +1,16 @@
+using System.ComponentModel.DataAnnotations;
```

Comment:
If this is part of the rich text validation, shouldn't this be added to the SecureRichTextAttribute?

Reply:
Not all rich text components are mandatory :-) So I think it's nice to keep the validation attributes separate.

Reply:
That probably works then. What is this doing differently from the normal [Required] attribute, client side validation?

Reply:
This one forces JQuery Validation to use the 'richtextrequired' client side validator instead if it's own default validator. The difference with the client side validator is that checks for actual content. 

For example: when pressing enter in the rich text editor, you'll get the following content: `<p></p>`. Which is valid content according to the default validator.

The rich text validator strips away the HTML attributes (thanks to `$(value).text().trim()`) and then checks if there is content.